### DISCUSSION TITLE
exposing convert to allow in memory usage (not only paths)

### DISCUSSION THREAD

**Discussion by framp** [visioncortex/vtracer#64] [Ideas]
I see in master a convert function which allow you to implement your own reading input and writing output but it doesn't seem like it's in 0.6.2 published con crates.io (while on github I see 0.6.1)

That would be a very welcome addition to implement vtracer in a pipeline of transformations

**Comment by tyt2y3** [comment]
wait, I didn't publish?

**Comment by tyt2y3** [comment]
I published now https://crates.io/crates/vtracer/0.6.3

**Comment by framp** [comment]
amazing, thanks!
